types/luxon/luxon-tests.ts

```diff
@@ -27,21 +27,26 @@ const fromObject = DateTime.fromObject({
 });
 
 const ianaZone = new IANAZone('America/Los_Angeles');
-const testIanaZone = IANAZone.create("Europe/London");
-IANAZone.isValidSpecifier("Europe/London");
-IANAZone.isValidZone("Europe/London");
+const testIanaZone = IANAZone.create('Europe/London');
+IANAZone.isValidSpecifier('Europe/London');
+IANAZone.isValidZone('Europe/London');
 IANAZone.resetCache();
-testIanaZone.formatOffset(dt.toMillis(), 'narrow'); // => +1
-testIanaZone.formatOffset(dt.toMillis(), 'short'); // => +01:00
-testIanaZone.formatOffset(dt.toMillis(), 'techie'); // => +0100
+testIanaZone.formatOffset(dt.toMillis(), 'narrow'); // $ExpectType string
+testIanaZone.formatOffset(dt.toMillis(), 'short'); // $ExpectType string
+testIanaZone.formatOffset(dt.toMillis(), 'techie'); // $ExpectType string
+testIanaZone.offsetName(dt.toMillis()); // $ExpectType string
```

Comment:
Why no options parameter here? Either you pass the param or use `$ExpectError`, since calling` offsetName` without options throws `Uncaught TypeError: Cannot read property 'format' of undefined`

Reply:
This is a weird case. Technically only Local and IANA zones need/use the options. I think that's the reason why it's marked optional in the types. That being said all the zones are meant to share a common interface. 
I think it's sane case to make that `options` on [this line](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8a6de885da2ebbb895985eea03f385a84cad7eaa/types/luxon/index.d.ts#L503) should not be optional.